modules/virt-updating-imported-vm-network-name.adoc

```diff
@@ -3,20 +3,20 @@
 // * virt/virtual_machines/importing_vms/virt-importing-vmware-vm.adoc
 
 [id="virt-updating-imported-vmware-vm-network-name_{context}"]
-= Updating the imported VMware virtual machine's NIC name
+= Updating the NIC name
 
 You must update the NIC name of a virtual machine imported from VMware to conform to {VirtProductName} naming conventions.
 
 .Procedure
 
 . Log in to the virtual machine.
-. Go to the `/etc/sysconfig/network-scripts` directory.
-. Change the network configuration file name to `ifcfg-eth0`:
+. Navigate to the `/etc/sysconfig/network-scripts` directory.
+. Rename the network configuration file:
 +
 ----
 $ mv vmnic0 ifcfg-eth0 <1>
 ----
-<1> Additional network configuration files are numbered sequentially, for example, `ifcfg-eth1`, `ifcfg-eth2`.
+<1> The first network configuration file is called `ifcfg-eth0`. Additional network configuration files are numbered sequentially, for example, `ifcfg-eth1`, `ifcfg-eth2`.
```

Comment:
s/called/named
s/sequentially, for example,/sequentially. For example:

Reply:
First, yes. Second, no. I don't like sentence fragments. It's better this way.